docker/docker-compose.local.yml

```diff
@@ -94,6 +95,7 @@ services:
       AWS_SECRET_ACCESS_KEY: $AWS_SECRET_ACCESS_KEY
       ACTIONSTEP_CLIENT_ID: $ACTIONSTEP_CLIENT_ID
       ACTIONSTEP_CLIENT_SECRET: $ACTIONSTEP_CLIENT_SECRET
+      MAILCHIMP_API_KEY: $MAILCHIMP_API_KEY
 
```

Comment:
You will need to add a `MAILCHIMP_API_KEY` for the test container as well in this file, so the tests pass

You can put an empty string as the value, since we won't actually be connecting to mailchimp

Reply:
done